### PR TITLE
added a question in freecodecamp-quiz

### PR DESCRIPTION
**Pull request by lakshyabhasin509**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/freecodecamp-quiz.ts [hunk 1]
```diff
@@ -106,6 +106,16 @@ const freecodecampQuiz = [
           "All of these projects have been a source of inspiration, but Wikipedia is the closest analog to what the freeCodeCamp community would ultimately like to become: hundreds of languages represented, with thousands of contributors from a wide variety of backgrounds and interests.",
         Link: "https://www.freecodecamp.org/news/welcome-to-the-abundance-economy-there-are-free-lunches-all-over-the-place-b9d0a417fd1a/",
       },
+      {
+        Question:
+          "Who is the founder of freeCodeCamp",
```
Comment (jdwilkin4) [inline src/data/freecodecamp-quiz.ts:111]:
```suggestion
          "Who is the founder of freeCodeCamp?",
```

File: src/data/freecodecamp-quiz.ts [hunk 2]
```diff
@@ -106,6 +106,16 @@ const freecodecampQuiz = [
...
+        Answer: "Quincy Larson",
+        Distractor1: "Madison Kanna",
+        Distractor2: "James Altucher",
+        Distractor3: "Navin Kabra",
+        Explanation:
+          "freeCodeCamp was launched in October 2014 and incorporated as Free Code Camp, Inc. The founder, Quincy Larson, is a software developer who took up programming after graduate school and created freeCodeCamp as a way to streamline a student's progress from beginner to being job-ready.",
```
Comment (jdwilkin4) [inline src/data/freecodecamp-quiz.ts:117]: Please do not use direct quotes from 3rd party resources. It is always best to write the explanations in your own words. 👍 

```suggestion
          "Quincy Larson founded freeCodeCamp in 2014 as a program meant to help individuals learn the basics of programming and land a developer job.",
```

File: src/data/freecodecamp-quiz.ts [hunk 3]
```diff
@@ -106,6 +106,16 @@ const freecodecampQuiz = [
...
+        Link: "https://www.freecodecamp.org/news/about/"},
```
Comment (jdwilkin4) [inline src/data/freecodecamp-quiz.ts:118]: Please properly format this object by placing the ending `}` on the next line. 👍

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

File: src/data/freecodecamp-quiz.ts (hunk 3 above)
Comment (lakshyabhasin509) [reply src/data/freecodecamp-quiz.ts:118]: Yeah , I'll keep that in mind.

**Comment by lakshyabhasin509** [comment]
I Did the suggested changes.

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**